ui/app/src/Utils/Date.elm

```diff
@@ -62,14 +65,19 @@ durationFormat time =
         |> String.trim
 
 
-dateFormat : ISO8601.Time -> String
```

Comment:
For sure. I would prefer to use [rluiten/elm-date-extra](https://github.com/rluiten/elm-date-extra) over [jweir/elm-iso8601](https://github.com/jweir/elm-iso8601) as it offers more string formatting functionality. One issue for me was to have leading zeroes on single digit hours.

@stuartnelson3 What do you think?

If you are both fine with the first one I would replace the current one entirely with the new one.